HUD/SetMinimapComponent.md

```diff
@@ -5,24 +5,38 @@ ns: HUD
 
 ```c
 // 0x75A9A10948D1DEA6 0x419DCDC4
-Any SET_MINIMAP_COMPONENT(int componentID, BOOL toggle, int hudColor);
+int SET_MINIMAP_COMPONENT(int componentID, BOOL toggle, int hudColor);
```

Comment:
I guess it's OK to change this; `Any` still resolves to `int` though, during csharp [codegen](https://github.com/citizenfx/fivem/blob/bb270ff5cb320831d77a2ca4541159a1dc582362/ext/natives/codegen_out_cs.lua#L571-L572C1).